nsxt/resource_nsxt_policy_vni_pool_test.go

```diff
@@ -0,0 +1,169 @@
+/* Copyright © 2020 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var accTestPolicyVniPoolCreateAttributes = map[string]string{
+	"display_name": getAccTestResourceName(),
+	"description":  "terraform created",
+	"start":        "75001",
+	"end":          "75100",
+}
+
+var accTestPolicyVniPoolUpdateAttributes = map[string]string{
+	"display_name": getAccTestResourceName(),
+	"description":  "terraform updated",
+	"start":        "75101",
+	"end":          "75200",
+}
+
+func TestAccResourceNsxtPolicyVniPool_basic(t *testing.T) {
+	testResourceName := "nsxt_policy_vni_pool.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
```

Comment:
This test should only run for Local Manager - `testAccOnlyLocalManager(t)`

Reply:
Done.